aea/connections/stub/connection.py

```diff
@@ -79,6 +84,25 @@ def _decode(e: bytes, separator: bytes = SEPARATOR):
 
     return Envelope(to=to, sender=sender, protocol_id=protocol_id, message=message)
 
+def _lock_fd(fd: IO[AnyStr]) -> bool:
```

Comment:
@lrahmani please don't use abbreviations like `fd` as they make it harder to read for other devs. Use full names where possible.